File: TransCoder_tokenized_test_set_functions/cpp/MINIMUM_OPERATION_MAKE_ELEMENTS_EQUAL_ARRAY.cpp
int minOperation ( int arr [ ], int n ) {
  unordered_map < int, int > hash;
  for ( int i = 0;
  i < n;
  i ++ ) hash [ arr [ i ] ] ++;
  int max_count = 0;
  for ( auto i : hash ) if ( max_count < i . second ) max_count = i . second;
  return ( n - max_count );
}

File: results/java_codex/MINIMUM_OPERATION_MAKE_ELEMENTS_EQUAL_ARRAY.java
int minOperation(int arr[], int n) {
        unordered_map<int, int> hash;
        for (int i = 0; i < n; i++)
            if (hash.find(arr[i]) != hash.end())
                hash[arr[i]]++;
            else
                hash[arr[i]] = 1;
        int max_count = 0;
        for (auto i : hash)
            if (max_count < i.second)
                max_count = i.second;
        return (n - max_count);
    }